src/games/tic_tac_toe.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+exports.setUp = function setUp(players) {
+  let player_sides = {
```

Comment:
I meant that maybe we want to provide support for constants held within the state. Or would we just be letting people using immutable.js?